### PR TITLE
Add v1.0 specific rules

### PR DESCRIPTION
**Pull request by johntran-nv**
Adding rules that we've agreed on the last few SWG meetings.

### CODE REVIEW

**Comment by nvcforster** [comment]
Suggested BERT v1.0 clip-norm rule edit:
For v1.0 only, BERT submissions may implement clip-norm either before or after inter-accelerator all-reduce. For future rounds, the expectation is that submissions must use clip-norm-after-reduce, to be consistent with most commonly used public BERT model repos. 
For performance consistency of at scale BERT submissions for v1.0, submitters are disallowed from using clip-norm-after-reduce to enable additional overlap of communication and math. If a submitter plans to use clip-norm-after-reduce for v1.0, they must notify the committee before the submission deadline, and be prepared to show code in their submission proving that they do not do overlap as a result of clip-norm-after-reduce.

File: training_rules.adoc [hunk 1]
```diff
@@ -521,4 +523,20 @@ Analysis to support this can be found in the document "MLPerf Optimizer Review"
...
 |      |              	          | MXNet	    | mx.optimizer.NAG
+|===
+
+== Appendix: v1.0 Specific Rules
+
+This section contains rules specific to the v1.0 round of MLPerf Training.  These do not apply to future rounds, unless explicitly ratified as rules for those rounds, or unless these rules are promoted to official rules in previous sections of this document.
+
+=== MaskRCNN backbone
+
+For v1.0 only, Mask-RCNN submitters may use the non-reference backbone located https://drive.google.com/drive/folders/1lGU_pP2Pr2k578DslciwE4AlOKm6dKCK?usp=sharing[here] with the understanding that it converges similarly to the reference backbone.  If the non-reference backbone is shown to converge differently than the reference backbone at any scale on any submitted hyperparameter set, all uses of that backbone for any submitter are to be re-run with the reference backbone to have their submission published.  For future rounds, the expectation is that all submitters will use the reference backbone, which will fixed at reference code freeze time.
```
Comment (emizan76) [inline training_rules.adoc]: Converge differently: Isn't it better to say "converge faster in terms of epochs"?

Comment (johntran-nv) [reply training_rules.adoc]: Hi Elias, I don't feel strongly on this one, but my thought was that if it is shown to converge slower or faster, then that is evidence that the backbone affects convergence, in which case we would conservatively want to disallow it.  What do you think of that?

Comment (emizan76) [reply training_rules.adoc]: I thought we'd have it go through RCP and if the non-ref backbone proved faster then we look into it. If not then we accepted as it is. Anyway, we can go through it one last time if needed.

**Comment by johntran-nv** [comment]
@emizan76 , any other comments on this one?  I fixed up some conflicts with master, so this should be ready to merge now.

**Comment by emizan76** [comment]
I wanted to discuss the wording or the meaning of the MaskRCNN convergence
with different backbones.
Actually it was one of the bullet points I wanted to bring up in the
previous meeting.
We think that if a submission with a different backbone passes the RCP
check then it should not be treated differently, even if it converges
differently (which in this case is more slowly).

Let me know what you think, if it urgent to have this merged, we can merge
it now and discuss it next week,

Elias

**Comment by johntran-nv** [comment]
@emizan76 , like I said, I don't feel strongly, so went ahead and changed it to "faster" as you suggest.  With that, are you ok with merging?

**Review by emizan76 — APPROVED** [review]

**Pull request merged**